src/core_plugins/kibana/public/management/sections/indices/_create.js

```diff
@@ -24,7 +24,7 @@ uiModules.get('apps/management')
     name: config.get('indexPattern:placeholder'),
     isTimeBased: true,
     nameIsPattern: false,
-    notExpandable: false,
+    expandable: false,
```

Comment:
Ah, I see.  I misread where this code was actually used.  Thanks for the clarification.